# CSC687 Homework #7 -- Paper Review

 * On paper: [Comprehensive Mapping of Long-Range Interactions Reveals Folding Principles of the Human Genome][1]
 * By George J. Pavelka on March 11th, 2020

_The paper describes a new method called Hi-C to map 3D architecture of whole genomes. Illustrated on human genome by coupling DNA sequences that are close to each other constructing spatial proximity maps (heat maps) at a resolution of 1 megabase. Being able to show areas of chromosomes and to give approximation of small chromosomes that are rich on genes._

The importance of understanding how chromosomes fold provides deeper insight of their distribution in chromatin, gene activity and complete cell functional state. Introduced method of Hi-C is in compliance with known methods of 3C, 4C and 5C and extends them into an unbiased analysis of the whole genome.

## Library of Read Pairs

_The goal of the process is to create library of spatially adjacent pairs of DNA sequences uniquely aligned to the human genome reference sequence. The library is then analyzed to produce a catalog of interacting DNA fragments._

 1. In order to obtain pairs of neighboring DNA sequences, first DNA must be cross-linked using formaldehyde.
 2. Using restriction enzyme, DNA is afterwards cut (digested) leaving short DNA overhanging.
 3. Loose DNA cuts are filled and marked with biotin resulting into their ligation between the neighboring (cross-linked) DNA fragments.
 4. DNA is then purified and sheared to isolate connected fragments.
 5. The remaining DNA sample contains fragments that were originally spatially close to each other. They are identified by paired-end sequencing.
 6. To create the library, only long-range read pairs (more than 20 kilobase apart) were used leaving 6.7 million fragments out of 8.4 million read pairs.

## Contact Matrix and Heat Map

_Contact matrix visualized as a heat map was constructed by sequential analysis of the library. The experiment was reproduced with same and different enzymes resulting in extremely similar matrices to the original one. Resulting heat map proved average intrachromosomal and interchromosomal principles in compliance to known methods._

Contact matrix was created by dividing the genome into 1 megabase chunks (loci). E.g. chromosome 14 contact matrix has 107 rows times 107 columns, because it has 107 million base pairs. Value of each cell corresponds to an amount of read pairs within given locus. From the matrix a heat map was generated simply as a color intensity given by a contact frequency. 

Above that, an average contact probability within chromosomes (intrachromosomal) was calculated. It showed a monotonic decrease in contact probability with chromosomal distance in a logarithmic progress. The increase in 3D distance between loci with increasing genomic distance is in compliance with known methods (3C and FISH). Measured probability is always greater than the average, even at long distances (over 200 mb). That implies existence of chromosome territories. 

It has been shown that chromosomes rich on genes tend to interact with each other more often compared to sparse ones. It is again in compliance with known method (FISH) that shows, that such chromosomes tend to appear near center of the nucleus. Including the small yet sparse chromosome 18 that does not interact and appears rather in the nuclear periphery.

## Compartments of Normalized Contact Matrix

_Normalization of contact matrix creates plaid pattern that can partition the whole genome into two spatial compartments. Regions within these compartments tend to be spatially closer to each other._

Contact matrices are strongly influenced by the sequential distance interactions of genomes. In order to visualize and locate the compartments, the average genome-wide contact probability has been removed from them. 

To sharpen the resulting plaid pattern, Pearson correlation between each row and column has been applied. That was based on an assumption, that neighboring loci have correlated interaction profiles. It was shown that up to 71% of the resulting matrix is highly correlated. 

The normalized matrix was decomposed into two sets of loci using principal component analyses such that contacts within each set are denser than between them. Additionally, the plaid patterns were consistent within each chromosome across chromosomes making the whole genome be divisible into two spatial compartments such that more interactions occur within them rather than across. 

Therefore, regions belonging to the same compartment are spatially closer to each other. It was tested using 3D-FISH on four sequential loci altering between both compartments. The test proved two loci being spatially closer to each other despite there was another locus sequentially in between of them. 

[1]: https://science.sciencemag.org/content/326/5950/289
(Science  09 Oct 2009: Vol. 326, Issue 5950, pp. 289-293, DOI: 10.1126/science.1181369u)
